FirebaseAppCheck/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- [changed] Removed usages of user defaults API to reduce required reason impact.
```

Comment:
Can we change "reduce" to "eliminate" throughout?

Reply:
Done in [1073c4e](https://github.com/firebase/firebase-ios-sdk/pull/12774/commits/1073c4ec28d6d857c322162adfed36440d7e6c58)